Source/Pages/Gallery/LibraryMediaManager.swift

```diff
@@ -210,7 +210,15 @@ open class LibraryMediaManager {
                 // 5. Configuring export session
                 let videoIsCropped = cropRect.size.width < abs(videoSize.width) || cropRect.size.height < abs(videoSize.height)
 
-                let presetName = compressionTypeOverride ?? (videoIsCropped || videoIsTrimmed || videoIsRotated || (shouldMute && videoNeedsProcessing) ? YPConfig.video.compression : AVAssetExportPresetPassthrough)
+                var presetName = ""
```

Comment:
Just a nit, this could be `let presetName: String` since it always gets inited in if-else.

Reply:
@MikkoKuivanenRS Updated https://github.com/rewardStyle/YPImagePicker/pull/44/commits/82209d18a3dac257172664a775fc74e4bcf3c81d